packages/canary-features/addon/default-features.ts

```diff
@@ -173,7 +173,6 @@ export default {
   SAMPLE_FEATURE_FLAG: null,
   RECORD_DATA_ERRORS: null,
   RECORD_DATA_STATE: null,
-  IDENTIFIERS: true,
```

Comment:
I think the practice is to keep this right? For example, removing it would break downstream code that is checking whether it is enabled

Reply:
Is that the practice? I’d think downstream would treat it as undefined

Reply:
In which case it would revert to the pre identifiers behavior, no?

Reply:
If wave if(Identifiers) in a m3 version, it ought to continue to work, no? Don't think there's a large cost in keeping. Maybe @rwjblue had thoughts

Reply:
Seems like a fine thing just unfamiliar with what we usually do here. Happy to keep if that’s best.